src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_BOOK_STATUS = 'CHECK_BOOK_STATUS';
+
```

Comment:
- [ ] As it is a requirement for this project, the value of each action type needs to follow the [ducks pattern](needSupport), please kindly fix that.